config/FCChh_DelphesCard_WithDipole_v00.tcl

```diff
@@ -37,7 +37,7 @@ TowerMerger
 EFlowMerger
 
 PhotonEfficiency
-PhotonEnergySmearing
+#PhotonEnergySmearing

```

Comment:
Ah, thanks. Can you remove it (or uncomment it, doesn't really matter) or should I make a new pull request?